openedx/core/djangoapps/cors_csrf/tests/test_middleware.py

```diff
@@ -263,15 +264,15 @@ def _assert_cookie_sent(self, response, is_set):
         """Check that the cross-domain CSRF cookie was sent. """
         if is_set:
             self.assertIn(self.COOKIE_NAME, response.cookies)
-            cookie_header = str(response.cookies[self.COOKIE_NAME])
+            cookie_header = six.text_type(response.cookies[self.COOKIE_NAME])
 
-            expected = b'Set-Cookie: {name}={value}; Domain={domain};'.format(
+            expected = 'Set-Cookie: {name}={value}; Domain={domain};'.format(
```

Comment:
This should be a unicode string.  You should prefix with a `u` so that it's explicitly unicode in python 2.  Otherwise it will be a byte string in python2 and a unicode string in python 3.  In the case of this test, that doesn't change behavior but in other places it might do that so be explicit when you want a string to be unicode for now.